README.md

```diff
@@ -1,160 +1,90 @@
-[jStat](http://www.jstat.org/) - JavaScript Statistical Library
+**jStat** - JavaScript Statistical Library
```

Comment:
```suggestion
**@handsontable/jstat** - JavaScript Statistical Library
```

To make it obvious in the first line that this is our fork, not the jStat library.